pandas/tests/io/sas/test_sas7bdat.py

```diff
@@ -174,3 +176,9 @@ def test_date_time():
     df0 = pd.read_csv(fname, parse_dates=['Date1', 'Date2', 'DateTime',
                                           'DateTimeHi', 'Taiw'])
     tm.assert_frame_equal(df, df0)
+
+def test_zero_variables():
+    dirpath = tm.get_data_path()
+    fname = os.path.join(dirpath, "zero_variables.sas7bdat")
+    with pytest.raises(EmptyDataError):
```

Comment:
can you test with the zero observations file as well (is it the same result)? you can parametrize if that'st the case

Reply:
The zero observations file gives a `None` value while reading an empty the csv file gives an empty dataframe. I think this is a bug but have not fixed it right now